build/BundledVersions.targets

```diff
@@ -0,0 +1,72 @@
+<Project>
+  <PropertyGroup>
+    <BundledVersionsIntermediateDirectory>$(IntermediateDirectory)/GeneratedMSBuildImports</BundledVersionsIntermediateDirectory>
+    <BundledVersionsPropsFolder>15.0/Imports/Microsoft.Common.props/ImportBefore</BundledVersionsPropsFolder>
+    <BundledVersionsPropsFileName>Microsoft.NETCoreSdk.BundledVersions.props</BundledVersionsPropsFileName>
+  </PropertyGroup>
+
+  <Target Name="GenerateBundledVersionsProps"
+          BeforeTargets="PublishMSBuildExtensions">
+
+    <!-- 
+      The bundled version of Microsoft.NETCore.App is determined statically from DependencyVersions.props whereas the bundled version
+      of NETStandard.Library is defined by what Microsoft.NETCore.App pulls in. This digs in to the package resolution items of
+      of our build against Microsoft.NETCore.App to find the correct NETStandard.Library version
+
+      NOTE: Package dependency items do not have package version or package id metadata, but their identities are [id]/[version].
```

Comment:
Another way to do this, that is more straight-forward IMO, would be to use `@(PackageDefinitions)`.

```
                         PackageDefinitions=
                             Microsoft.NETCore.App/2.0.0-beta-001820-00
                                     Name=Microsoft.NETCore.App
                                     Path=microsoft.netcore.app/2.0.0-beta-001820-00
                                     ResolvedPath=C:\Users\eerhardt\.nuget\packages\microsoft.netcore.app\2.0.0-beta-001820-00
                                     Type=package
                                     Version=2.0.0-beta-001820-00
                             Microsoft.NETCore.DotNetAppHost/2.0.0-beta-001820-00
                                     Name=Microsoft.NETCore.DotNetAppHost
                                     Path=microsoft.netcore.dotnetapphost/2.0.0-beta-001820-00
                                     ResolvedPath=C:\Users\eerhardt\.nuget\packages\microsoft.netcore.dotnetapphost\2.0.0-beta-001820-00
                                     Type=package
                                     Version=2.0.0-beta-001820-00
                             Microsoft.NETCore.Platforms/2.0.0-beta-25121-03
                                     Name=Microsoft.NETCore.Platforms
                                     Path=microsoft.netcore.platforms/2.0.0-beta-25121-03
                                     ResolvedPath=C:\Users\eerhardt\.nuget\packages\microsoft.netcore.platforms\2.0.0-beta-25121-03
                                     Type=package
                                     Version=2.0.0-beta-25121-03
                             Microsoft.Packaging.Tools/1.0.0-beta-25121-01
                                     Name=Microsoft.Packaging.Tools
                                     Path=microsoft.packaging.tools/1.0.0-beta-25121-01
                                     ResolvedPath=C:\Users\eerhardt\.nuget\packages\microsoft.packaging.tools\1.0.0-beta-25121-01
                                     Type=package
                                     Version=1.0.0-beta-25121-01
                             NETStandard.Library/2.0.0-beta-25121-01
                                     Name=NETStandard.Library
                                     Path=netstandard.library/2.0.0-beta-25121-01
                                     ResolvedPath=C:\Users\eerhardt\.nuget\packages\netstandard.library\2.0.0-beta-25121-01
                                     Type=package
                                     Version=2.0.0-beta-25121-01
```

Then you could base the conditions and include on `%(Name)` and `%(Version).